### DISCUSSION TITLE
Add Django to Framework Guides

### DISCUSSION THREAD

**Discussion by BADARRASHEED** [tailwindlabs/tailwindcss#13773] [Ideas]
Dear Tailwind Labs, it is very easy to use tailwind with laravel as compared to **Django** please also add tailwind installation in django in Framework Guides just like laravel it would be a great helping hand from you😊. Please do consider this idea waiting for your reply

**Comment by BADARRASHEED** [comment]
![Screenshot 2024-06-01 014113](https://github.com/tailwindlabs/tailwindcss/assets/98841403/70bb1845-4564-419a-a3ba-4ba5177611db)


Yes I need Django Installation guide here please.

**Comment by paatre** [comment]
What have you been using for Tailwind setup in Django projects?

**Comment by EricE** [comment]
https://github.com/timonweb/django-tailwind
